### ISSUE TITLE
Conflict with W3 Total Cache WP plugin

### ISSUE THREAD

**Issue by zaccety** [stevenwanderski/bxslider-4#1166]
BX Slider fails in WordPress when enabling the JS Minify setting in W3 Total Cache, no console errors are thrown.

**Comment by ginsterbusch** [comment]
Have you already tested it with different caching solutions, eg. Cachify or Hypercache?

cu, w0lf.

**Comment by zaccety** [comment]
No, not yet, my preferred caching plugin is currently W3 but I can test with alternatives.

Will feedback here when checked.

**Comment by ginsterbusch** [comment]
W3 Total Cache has always been rather difficult and prone to errors, thats why I'm asking. My personal suggestion (as a long-term WordPress developer) is to use Hypercache or a derivate of it, because issues and configuration issues (read: "accidential misconfiguration thanks to crappy UI / bad documentation").

But it might be just something trivial, eg. W3 Total Cache trying to use the CDN version, too extreme minification (variable replacements might lead to unwanted side-effects), etc. If possible, I'd exclude bxslider from any kind of minification etc. processes (just serve the already minified version "as-is").

cu, w0lf.

**Comment by zaccety** [comment]
Thanks for the tip w0lf, I've been developing WP sites for quite a while now, but have to admit that the caching plugins still confuse the hell out of me at times.

I'll test your suggestions and update this thread with the results.

**Comment by zaccety** [comment]
Just to feedback on this, I tried Hypercache and it did indeed stop the conflict with the BX slider jQuery, however it dropped my page speed scores considerably.

After playing around with various combinations of other caching and minfication plugins (not exhaustively, who's got the time?) I've reverted to W3 Total Cache, but set the Minify Mode to manual - this allowed the BX Jquery to run unimpeded - and then used this in combination with the Fast Velocity Minify plugin, this gave me the best results so far according to Google Page Speed Insights, GT Metrix and WebPageTest.

It's a jungle out there.
